tests/test_define.h

```diff
@@ -107,8 +107,8 @@ extern "C" {
   "999999999999999999999999999"
 
 #define TEST_BUNDLE_HASH                                                       \
-  "QBFXQETKSHDYPFUDO9ILVCAVQIXOHXKCECZYFLPBNVIX9JUXQZJE9URQEEUWPWYZOIACTCGZX9" \
-  "IDIODCA "
+  "VPSUKRVIVHYFLJIJOKFAEWQYISIFODXXDNKBZRXHJTCIXJT9KWAWOLN9DLHTHGDQSFAWZBDWUJ" \
```

Comment:
Is it not associated to this pull request?

Reply:
This is more related to #146. I will change it once it's merged.